app/services/email.service.js

```diff
@@ -15,117 +15,103 @@ const connectorClient = function () {
   return new ConnectorClient(process.env.CONNECTOR_URL)
 }
 
-const getEmailSettings = function (accountID, correlationId) {
-  return new Promise(function (resolve, reject) {
-    const startTime = new Date()
-    connectorClient().getAccount({
+const getEmailSettings = async function (accountID, correlationId) {
+  const startTime = new Date()
+  try {
+    const data = await connectorClient().getAccount({
       gatewayAccountId: accountID,
       correlationId: correlationId
     })
-      .then(data => {
-        logger.info(`[${correlationId}] - GET account %s ended - elapsed time: %s ms`, accountID, new Date() - startTime)
-        resolve({
-          customEmailText: data.email_notifications.PAYMENT_CONFIRMED.template_body,
-          emailEnabled: data.email_notifications.PAYMENT_CONFIRMED.enabled,
-          emailCollectionMode: data.email_collection_mode,
-          refundEmailEnabled: data.email_notifications.REFUND_ISSUED && data.email_notifications.REFUND_ISSUED.enabled
-        })
-      })
-      .catch(err => {
-        logger.info(`[${correlationId}] - GET account %s ended - elapsed time: %s ms`, accountID, new Date() - startTime)
-        clientFailure(err, reject, 'GET', correlationId, false)
-      })
-  })
+    logger.info(`[${correlationId}] - GET account %s ended - elapsed time: %s ms`, accountID, new Date() - startTime)
```

Comment:
I imagine we can remove all the request logging from this service. The client does its own logging for requests. Might want to be sure it's not used by any dashboards I guess, but I doubt these ones are.